lib/DDG/Spice/DebVersion.pm

```diff
@@ -0,0 +1,21 @@
+# Get the version of a package in various debian repositories
+package DDG::Spice::DebVersion;
+
+use DDG::Spice;
+
+triggers any => "debian version", "debian versions", "deb ver", "debian ver";
```

Comment:
Probably best to use `startend` otherwise we'll trigger this on every query with these words in it.